Makefile

```diff
@@ -57,9 +59,10 @@ controller.image: docker/Dockerfile docker/controller
 	$(KUBECFG) show -V CONTROLLER_IMAGE=$(CONTROLLER_IMAGE) -o yaml $< > $@.tmp
 	mv $@.tmp $@
 
-controller.yaml: controller.jsonnet controller.image controller-norbac.jsonnet
```

Comment:
`controller.image` target breaks GoReleaser.
Remove from here but add to `.travis.yml` to make up for it.